_includes/docs/extending/downloading.md

```diff
@@ -0,0 +1,25 @@
+Depending on, if you have a composer, command-line, drush-make or UI based workflow, downloading of themes and modules differs.
+
+### Composer
+Composer is a package manager for PHP and is the future standard for managing the dependencies of your Drupal 8 site and extensions. Go to the [Drupal documentation](https://www.drupal.org/docs/develop/using-composer/using-composer-with-drupal) for more information.
+
+If you have composer installed, go to the root of your site (here there should be a `composer.json` file) and install modules by typing `composer require drupal/[shortname of module]` into the command line.  
+For example: `composer require drupal/entity_browser`  
```

Comment:
We should use same formatting for commands in our documentation. I have used:
```
# my_command
```
So it's always new line, way more distinguish from text and ready to copy paste. I would like to have one day something like this: https://www.tensorflow.org/tutorials/mandelbrot (so you can change code theme and also copy it). That would be nice. ;)

But until then, we just should think about best way to format command lines and use same formatting in documentation.